### DISCUSSION TITLE
[Feature Request] QInput Rules

### DISCUSSION THREAD

**Discussion by arnonrdp** [quasarframework/quasar#14896] [Ideas / Proposals]
I would like to suggest an improvement for when we are using the `rules` attribute.

In this example: https://quasar.dev/vue-components/input#example--form-validation
I believe it would be interesting for the validation to be reset if the user clears the input field.

Currently, I'm using trickery and doing the following:
`@blur="!modelValue ? $refs.modelValue.resetValidation() : $refs.modelValue.validate()"`

But it would be really cool if it was native.

**Comment by yusufkandemir** [comment]
Can you share your use case, please? What's the field used for, what are the validation rules, and why do you need to reset it after the user clears the input field?

By the way, an alternative trickery with a different behavior would be listening to `@update:model-value` to reset if needed, it will be validated as usual otherwise(_no need to call validate()_).

**Reply by arnonrdp** [reply]
It is a simple login and password form.

The problem is that the user doesn't want to see an error if he erase all characters. That's all, just makeup.

Regarding `$refs.modelValue.validate()` you are right, it was redundant to call it, thanks for that.
